client/sites-dashboard/components/sites-content-controls.tsx

```diff
@@ -135,12 +135,24 @@ export const SitesContentControls = ( {
 					selectedText={ sprintf( __( 'Status: %(siteStatus)s' ), {
 						siteStatus: selectedStatus.title,
 					} ) }
+					ariaLabel={ sprintf(
+						// Translators: `siteStatus` is one of the site statuses specified in the Sites page.
+						__( 'Filtering by Status "%(siteStatus)s". Switch status filter.' ),
+						{
+							siteStatus: selectedStatus.title,
+						}
+					) }
 				>
 					{ statuses.map( ( { name, title, count } ) => (
 						<SelectDropdown.Item
 							key={ name }
 							selected={ name === selectedStatus.name }
 							count={ count }
+							// Translators: `siteStatus` is one of the site statuses specified in the Sites page. `count` is a number of sites of given status.
+							ariaLabel={ sprintf( __( '%(siteStatus)s (%(count)d sites)' ), {
```

Comment:
Does this format make sense for translations? Is it clear?

Reply:
My (limited) understanding of translations is that it's better to have more distinct/explicit strings for each option, e.g.:
* `'Private (%(count)d sites)'`
* `'Coming soon (%(count)d sites)'`

However, I think I'd personally prefer `%(count)d private sites` instead of using parens.

Additionally, I don't know whether it would be a good or bad thing to include a verb in the `aria-label` (e.g. `Filter to %(count)d private sites`). Maybe not, because it's an option in the list?

Reply:
@danielbachhuber the status is already being translated, and if we get the new label translated, we would have the  following for ES:

> Público (15 sitios)

What if we leave the current format and let the translator adjust it? Then we could use completely different format if given languages requires that e.g.:

> 15 sitios "públicos"

Reply:
> What if we leave the current format and let the translator adjust it?

@wojtekn Sure, that works for me !